setup

```diff
@@ -160,7 +160,7 @@ install() {
 
 # read command line arguments
 opts=$(getopt \
-  --longoptions "$(printf "help,%s:," "${CMDLINE_ARGUMENT_LIST[@]}")" \
+  --longoptions "$(printf "help,skip-systemctl,%s:," "${CMDLINE_ARGUMENT_LIST[@]}")" \
```

Comment:
Leo, could you please fix the option name here instead: https://github.com/containers/qm/blob/main/tests/e2e/run-test-e2e#L67 ?

Reply:
@odra actually is not fix the option name, we should add there.

Reply:
ok, i figure out. lets move forward what we have.